types/unit/service.pp

```diff
@@ -25,6 +25,9 @@
     Optional['ExecStopPost']              => Variant[Systemd::Unit::Service::Exec,Array[Systemd::Unit::Service::Exec,1]],
     Optional['KillSignal']                => Pattern[/^SIG[A-Z]+$/],
     Optional['KillMode']                  => Enum['control-group', 'mixed', 'process', 'none'],
+    Optional['Nice']                      => Variant[String[0,0],Integer[-20,19]],
```

Comment:
It took me a second to realize `String[0,0]` equals `Enum['']`. Thoughts on what should be preferred?

Reply:
* `Enum[''] when part of a  longer `Enum` clearly makes sense.
* `String[0,0]` reads very obviously as the empty string.

Expect `String[0,0]` is quicker though that's nothing.

Reply:
I've gotten used to writing `Enum['']` because I just never thought about `String[0,0]` before. In support of`Enum['']`: it's shorter.

I'm fine either way, just wanted to write down my thoughts here.